.idea/dictionaries/prodigy.xml

```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="prodigy">
+    <words>
+      <w>kitspace</w>
+    </words>
+  </dictionary>
+</component>
```

Comment:
Do we need to add all these `.idea` files? :worried:

Reply:
ok, just will gitignore them.